heidegger_index/models.py

```diff
@@ -12,6 +13,7 @@ class Work(models.Model):
     key = models.CharField(max_length=8, unique=True)
     csl_json = models.JSONField()
     reference = models.CharField(max_length=200, null=True)
+    reference_md = models.CharField(max_length=200, null=True)
```

Comment:
Cool dat je dit aan de models hebt kunnen toevoegen, maar ik zit me af te vragen of de markdown niet net zo goed on the fly gegenereerd kan worden in de view. Het is een extra regeltje Python code die volgens mij niet veel overhead zal geven (vergelijkbaar met de DB query die nu nodig is denk ik). Wat was jouw overweging?

Reply:
Ik denk dat dat prima kan. Ik weet niet goed wat wijsheid is. Zelf heb ik niet zoveel inzicht in hoeveel overhead dat veroorzaakt. 

Verder gebruik ik nu [markdownify](https://github.com/matthewwithanm/python-markdownify) om in het populate script de html reference om te zetten naar markdown. Dat kan natuurlijk ook prima in de view.

Reply:
Ik zag het ja, het was idd mijn suggestie die markdownify-regel in de view te zetten.

Ik denk dat de voornaamste reden was dat het niet helemaal lekker voelt om dezelfde data op twee manieren op te slaan. Over het algemeen is het altijd goed om één source of truth te hebben. Daar breek ik strikt gezien natuurlijk mee door de CSL-json en de HTML reference apart op te slaan, maar omdat die conversie op een externe API-call berust, kan dat wel een redelijk 'dure' conversie zijn.

In dit geval lijkt me dat niet zo, een Python script moet sowieso uitgevoerd worden om de reeks te genereren, en dat markdownify script op een in de regel korte HTML string lijkt me daarin verwaarloosbaar (hoewel we dat natuurlijk ook kunnen testen om daar zeker van te zijn). Wat de dubbele conversies betreft: een mooiere manier om dat te omzeilen (op ten duur) is [caching](https://docs.djangoproject.com/en/4.0/topics/cache/).

Reply:
Okee! Ik zal het om-schrijven. Op zich is het ook niet echt heel erg dat de export meer op zich zou laten wachten.
Caching lijkt me nu nog niet aan de orde, maar goed.